pkg/users/account-create-dialog.js

```diff
@@ -250,6 +263,12 @@ export function account_create_dialog(accounts) {
                                         user_name,
                                         "--lock"
                                     ], { superuser: "require", err: "message" });
+                                if (state.change_passw_force)
+                                    return cockpit.spawn([
+                                        "/usr/bin/passwd",
+                                        "-e",
+                                        state.user_name
```

Comment:
The code updated not to use these global objects, please don't use `state.user_name`.